src/server/controllers/user/resendVerifyEmail.js

```diff
@@ -0,0 +1,26 @@
+import { getUserByEmail } from '#handlers/userHelpers.js';
+import { sendVerificationEmail } from '#handlers/sendEmail.js';
+
+async function resendVerifyEmail(req, res, next) {
+  try {
+    const { email } = req.body;
+    if (!email) {
+      return res.status(400).json({ message: 'Missing required field email' });
+    }
+    const user = await getUserByEmail(email);
+    if (!user) {
+      return res.status(404).json({ message: 'User not found' });
+    }
+    if (user.verify) {
+      return res
+        .status(400)
+        .json({ message: 'Verification has already been passed' });
+    }
+    await sendVerificationEmail(user.email, user.verificationToken);
```

Comment:
Passing params with dots is not good idea. 
I suggest to desctuct user or create new variable and then pass it to our function like so
f. ex. :
`const { email = userEmail, verificationToken } = await getUserByEmail( email );`
...
`sendVerificationEmail( userEmail, verificationToken );

Reply:
❓

Reply:
I don't know what's wrong. I destructed object.